sdk/framework/openapi.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/sdk/version"
 	"github.com/mitchellh/mapstructure"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
```

Comment:
would this add a new dependency?
currently an indirect https://github.com/hashicorp/vault/blob/4ec2c6ab4f0286e7cb35ea8579a02d182839f9b9/go.mod#L414

Reply:
Hmm `go mod tidy` didn't seem to change anything 🤷. For context, `strings.Title` has been deprecated in favour of `golang.org/x/text/cases.Title` so I'm adding it here.

Reply:
ahhh know what, its in `sdk` which is a different module

Reply:
good catch